recipes/ectyper/meta.yaml

```diff
@@ -0,0 +1,30 @@
+about:
+    license: MIT
+    summary: Ectyper is a python script for serotyping e.coli genomes
+    author: Camille La Rose, Chad Laing, Sam Sung
+    home: https://github.com/phac-nml/ecoli_serotyping
+
+package:
+    name: ectyper
+    version: 0.0.8
+
+source:
+    url: https://github.com/phac-nml/ecoli_serotyping/archive/v0.1.tar.gz
+    md5: 06b5793ed0f7000a05e20600b1ac5d0c
+
+requirements:
+    build:
+        - python 3.6.3.*
```

Comment:
Python should not be pinned this way.
Is there any reason to skip python 3.5? Please skip all python version in the build/skip section.